descope/internal/mgmt/user.go

```diff
@@ -11,18 +11,22 @@ type user struct {
 }
 
 func (u *user) Create(loginID, email, phone, displayName string, roles []string, tenants []*descope.AssociatedTenant) (*descope.UserResponse, error) {
-	return u.create(loginID, email, phone, displayName, roles, tenants, false)
+	return u.create(loginID, email, phone, displayName, roles, tenants, false, false)
+}
+
+func (u *user) CreateForTest(loginID, email, phone, displayName string, roles []string, tenants []*descope.AssociatedTenant) (*descope.UserResponse, error) {
```

Comment:
I would call the function createTestUser

Reply:
Done